containerfiles/rocky-8.8

```diff
@@ -0,0 +1,33 @@
+# syntax=docker/dockerfile:1.4.3
+
+FROM docker.io/library/rockylinux:8.8
```

Comment:
The repos in this image just pull from the main RL8 mirror, so it's not necessarily going to stay at 8.8

```
[baseos]
name=Rocky Linux $releasever - BaseOS
mirrorlist=https://mirrors.rockylinux.org/mirrorlist?arch=$basearch&repo=BaseOS-$releasever
#baseurl=http://dl.rockylinux.org/$contentdir/$releasever/BaseOS/$basearch/os/
gpgcheck=1
enabled=1
countme=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-rockyofficial
```